### PR TITLE
[1LP][RFR] new test_tag_mapping_azure_instances

### PR DESCRIPTION
**Pull request by lina-nikiforova**
{{ pytest: -v cfme/tests/cloud/test_tag_mapping.py::test_tag_mapping_azure_instances --use-provider=complete }}

Here I automated the manual test `test_tag_mapping_azure_instances`. 
I'm planning to expand the tag mapping testing later.

### CODE REVIEW

File: cfme/tests/cloud/test_tag_mapping.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.exceptions import ItemNotFound
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+
+from widgetastic.utils import partial_match
+
+pytestmark = [
+    pytest.mark.provider([AzureProvider], selector=ONE_PER_CATEGORY, scope='module'),
```
Comment (sshveta) [inline cfme/tests/cloud/test_tag_mapping.py]: Scope of provider should be function

File: cfme/tests/cloud/test_tag_mapping.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                                                category='Testing')
+    yield tag
+    request.addfinalizer(lambda: tag.delete())
+
+
+@pytest.fixture(scope='function')
+def vm(appliance, provider):
+    collection = appliance.provider_based_collection(provider, coll_type='vms').all()
+    try:
+        return [entity for entity in collection if entity.name == 'tag_mapping_vm'][0]
```
Comment (sshveta) [inline cfme/tests/cloud/test_tag_mapping.py]: Are we maintaining this VM on azure for this test ?

File: cfme/tests/cloud/test_tag_mapping.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+def test_tag_mapping_azure_instances(vm, map_tags, refresh_provider):
+    """"
+    1. Find Instance that tagged with test:testing in Azure (cu-24x7)
+    2. Create tag mapping for Azure instances
+    3. Refresh Provider
+    4. Go to Summary of the Instance
+    Expected result: In Smart Management field should be:
+    My Company Tags Testing: testing
+
+    Polarion:
```
Comment (mshriver) [inline cfme/tests/cloud/test_tag_mapping.py]: :tada:

File: cfme/tests/cloud/test_tag_mapping.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+@pytest.fixture(scope='function')
+def refresh_provider(provider):
+    provider.refresh_provider_relationships()
+    wait_for(provider.is_refreshed, func_kwargs={'refresh_delta': 10}, timeout=600)
+    return True
+
+
+def test_tag_mapping_azure_instances(vm, map_tags, refresh_provider):
+    """"
+    1. Find Instance that tagged with test:testing in Azure (cu-24x7)
```
Comment (mshriver) [inline cfme/tests/cloud/test_tag_mapping.py]: You can put this in the `Polarion` block under `testSteps` if you would like.  Enumerated list, there are examples in #8261

**Pull request merged**